src/guide/migration/filters.md

```diff
@@ -75,17 +75,17 @@ Instead of using filters, we recommend replacing them with computed properties o
 
 ### Global Filters
 
-If you are using filters that were globally registered and then used throughout our app, it's likely not convenient to replace them with computed props or methods in each individual component.
+If you are using filters that were globally registered and then used throughout your app, it's likely not convenient to replace them with computed properties or methods in each individual component.
 
-Instead, you can make your global filters available to all components through `app.globalProperties`:
+Instead, you can make your global filters available to all components through [globalProperties](../../api/application-config.html#globalproperties):
 
 ```javascript
 // main.js
 const app = createApp(App)
 
 app.config.globalProperties.$filters = {
-  accountInUSD(num) {
-    return '$' + num
+  currencyUSD(value) {
```

Comment:
For consistency with the Vue 2 example, I've changed the names to `currencyUSD`, `value` and `accountBalance`.